docs/node-mixin/config.libsonnet

```diff
@@ -52,6 +52,13 @@
     fsSpaceAvailableCriticalThreshold: 5,
     fsSpaceAvailableWarningThreshold: 3,
 
-    grafana_prefix: '',
+
+    grafanaK8s: {
```

Comment:
Why `K8s`? This mixin might be used in a context where Kubernetes doesn't exist at all.

Reply:
Thanks . I fix